src/Application/Console/Definitions/DefaultDefinition.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\PhpInsights\Application\Console\Definitions;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputOption;
+
+/**
+ * Minimum definition for each of our commands.
+ *
+ * @internal
+ */
+final class DefaultDefinition
```

Comment:
Consider making it an abstract class with a method to override in `AnalyseDefinition` for adding additional definitions to the class. That method for additional definitions could just return an empty array as default in this abstract class.